spec/models/king_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe King, type: :model do
+  
+  describe "King's valid_move" do
+  	it "should be inside destination, blocked" do
```

Comment:
This description is pretty unclear. I would go with something along the lines of `"should properly evaluate blocked moves that are otherwise valid"`.